pkg/koordlet/runtimehooks/reconciler/reconciler.go

```diff
@@ -484,6 +484,31 @@ func (c *reconciler) reconcilePodCgroup(stopCh <-chan struct{}) {
 					}
 				}
 
+				for _, initContainerStat := range podMeta.Pod.Status.InitContainerStatuses {
```

Comment:
perhaps the completed/failed init containers should be filtered out, and sidecar containers should be handled

Reply:
we leave a TODO here